pages/linux/wodim.md

```diff
@@ -0,0 +1,17 @@
+# wodim
+
+> Command for record/burning data using a optical drive to CD or DVD.
+> Caution when using this command.
+> There are many ways this command can be destructive, the following below is meant as a reminder of wodims syntax and not as the only source of documentation.
+
+- Display optical drives avaliable to wodim:
+
+`wodim --devices`
+
+- Burn file to disc in optical drive:
+
+`wodim -eject -tao speed=0 dev=/dev/{{optical_drive}} -v -data /{{file.iso}}`
```

Comment:
We should probably mention what does speed=0 and speed=1 do, since you are changing the option in the next command.

Also, why is there a "/" before file.iso ? We can just write {{file.iso}}.